### DISCUSSION TITLE
Something wrong of form .rec of my own data

### DISCUSSION THREAD

**Discussion by larsonwu0220** [apache/incubator-mxnet#13053] [Q&A]
I try to form .rec and .idx of my own data.
First, run im2rec.py to generate list :
python im2rec.py --list --recursive --train-ratio 1 [list folder] [images folder]
![image](https://user-images.githubusercontent.com/19381177/47765477-1adbfd80-dd05-11e8-9adf-d629b65f009a.png)

Second, run im2rec.py to generate .rec and .idx.
python im2rec.py --num-thread 4 [list folder] [images folder]
![image](https://user-images.githubusercontent.com/19381177/47765592-a9e91580-dd05-11e8-9073-73c5fd039848.png)
![image](https://user-images.githubusercontent.com/19381177/47765605-b40b1400-dd05-11e8-9231-b63b68f1a9be.png)

But when I read index, 
```
imgrec = mx.recordio.MXIndexedRecordIO(args.idx_path, args.bin_path, 'r')
s = imgrec.read_idx(0)
header, _ = mx.recordio.unpack(s)
```
The "header" has no label value.

![image](https://user-images.githubusercontent.com/19381177/47765730-50cdb180-dd06-11e8-9107-2e3024c3ace6.png)

Where is the problem?
Thanks.

**Comment by frankfliu** [comment]
@mxnet-label-bot [Question]

**Comment by NRauschmayr** [comment]
Thanks a lot for reporting this issue. Is every label in the file incorrect or just a few?
